backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -52,45 +53,64 @@ void roundTrip() {
         _service.addPatient(null, "BAR", "Basil", null, "Barnacle", "4th", LocalDate.of(1865, 12, 25), "13 Main", null,
                 "Hicksville", "NY",
                 "11801", "5555555555", PersonRole.STAFF, null, "Nassau", null, null, null, false, false);
-        List<Person> all = _service.getPatients(null);
+        List<Person> all = _service.getAllPatients();
         assertEquals(2, all.size());
     }
 
     @Test
     @WithSimpleReportStandardUser
     void deletePatient_standardUser_error() {
-        Person p = _service.addPatient(null, "FOO", "Fred", null, "Fosbury", "Sr.", LocalDate.of(1865, 12, 25), "123 Main",
+        Facility fac = _dataFactory.createValidFacility(_orgService.getCurrentOrganization(), "First One");
+        UUID facilityId =fac.getInternalId();
+
+        Person p = _service.addPatient(facilityId, "FOO", "Fred", null, "Fosbury", "Sr.", LocalDate.of(1865, 12, 25), "123 Main",
                 "Apartment 3", "Hicksville", "NY",
                 "11801", "5555555555", PersonRole.STAFF, null, "Nassau", null, null, null, false, false);
 
         assertSecurityError(() -> _service.setIsDeleted(p.getInternalId(), true));
-        assertEquals("Fred", _service.getPatients(null).get(0).getFirstName());
+        assertEquals("Fred", _service.getAllPatients().get(0).getFirstName());
+
+        assertSecurityError(() -> _service.getAllPatientsInclDeleted());
```

Comment:
These should be in a separate test, I think? They aren't actually testing the thing it says in the test name.

(That test could literally just be these two lines, though.)